.github/workflows/repository-visualizer.yml

```diff
@@ -0,0 +1,17 @@
+name: Create repository visualisation diagram
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - main
+jobs:
+  get_data:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Update diagram
+        uses: githubocto/repo-visualizer@main

```

Comment:
Can change the tag from main to 0.7.1? This will limit risk of this breaking in the future.

Reply:
It's done!